### PR TITLE
DHAS 2 to 12

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Inventory of Enda Abuna Samuel, up to no. 12.

### CODE REVIEW

File: DabraHalleloEndaAbunaSamuel/DHAS006.xml [hunk 1]
```diff
@@ -123,15 +123,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                               <desc>Regular, careful; by a well-trained scribe.</desc>
                            </handNote>                           
                        </handDesc>      
                                  
                                  <additions>
                                      <list>
                                          <item xml:id="a1">
                                              <locus target="#69r"/>
-                                             <desc type="Unclear">A title or indiction written in the upper margin, in red bold letters</desc>
+                                             <desc type="Unclear">A title or an indication is written in the upper margin, in red bold letters</desc>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS006.xml:134]: End this sentence with a period.

File: DabraHalleloEndaAbunaSamuel/DHAS007.xml [hunk 2]
```diff
@@ -156,8 +156,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                
                                <item xml:id="a6">
                                    <locus target="#181rb"/>
-                                   <desc type="Unclear">A commissioner's note mentioning  ʾabbā Gabra Dǝngǝl, the  called the bishop of the sanctuary of ʾabbā Sāmuʾel 
-                                       on</desc> 
+                                   <desc type="Unclear">A commissioner's note mentioning  ʾabbā Gabra Dǝngǝl, here 
+                                       called the bishop (ኤጲስ፡ ቆጶስ፡) of the sanctuary of ʾabbā Sāmuʾel of Halle Luya</desc>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS007.xml:160]: Luyā ?

File: DabraHalleloEndaAbunaSamuel/DHAS008.xml [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                                    <desc>Fine hand.</desc>
+                                </handNote>
+                            </handDesc>                            
+                                                             
+                                                        
+                        </physDesc>
+                        
+                        <history>
+                            <origin>
+                                <origDate notBefore="1600" notAfter="1799" source="lettering">17th century</origDate>.
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS008.xml:207]: notAfter="1699" or 17th/18th century

File: DabraHalleloEndaAbunaSamuel/DHAS009.xml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1700" notAfter="1799" source="lettering">17th/18th century</origDate>.
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS009.xml:87]: notBefore="1600"

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 5]
```diff
@@ -0,0 +1,176 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DHAS011" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gate of Lite, Covenant of the Morgen, Mǝʿrāf-chants</title>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:8]: light, morning

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+                        
+                        <msItem xml:id="ms_i1">       
+                            <locus from="9vb"/>
+                            <title type="complete" ref="LIT1113Anqasa">Gate of Light</title>
+                            <textLang mainLang="gez"/>                         
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">     
+                            <locus from="27r"/>
+                            <title type="complete" ref="LIT1716Kidanz">Covenant of the Morgen</title>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:42]: morning

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+                                       </support>
+                                       <extent>
+                                           <measure unit="leaf" quantity="132"/>                                            
+                                           <dimensions type="outer">
+                                               <height unit="mm" cert="medium">270</height>
+                                               <width unit="mm" cert="medium">190</width>
+                                               <depth unit="mm" cert="medium">70</depth>
+                                           </dimensions>
+                                       </extent>   
+                            <condition key="deficient">The binding is worn. One of the boards is spilt, textile inlays missing; sewing damaged.</condition>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:72]: split

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+                       <additions>
+                           <list>
+                               <item xml:id="a1">
+                                   <locus target="#35va"/>
+                                   <desc type="Unclear">A short excerpt from a liturgical text</desc>
+                               </item>
+                               
+                               <item xml:id="a2">
+                                   <locus target="#131va"/>/>
+                                   <desc type="MagicText">A short text with ʾasmāt, half-erased</desc>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:107]: `<foreign xml:lang="gez">ʾasmāt<foreign>`

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1700" notAfter="1799">Possibly second half of the 18th century</origDate>.
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:133]: notBefore="1750"

File: DabraHalleloEndaAbunaSamuel/DHAS011.xml [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1700" notAfter="1799">Possibly second half of the 18th century</origDate>. 
+                             The name of <persName ref="PRS10450YosabI">Metropolitan Yosāb (II)</persName> is mentioned 
+                             at the end (<locus target="#131rb"/>); the name of ʾAskāla Māryām is mentioned in
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS011.xml:135]: <persName ref="PRS00000000000">ʾAskāla Māryām</persName>

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+                                           <measure unit="leaf" quantity="156">156</measure>                                         
+                                                                               
+                                           <dimensions type="outer">
+                                               <height unit="mm" cert="medium">220</height>
+                                               <width unit="mm" cert="medium">210</width>
+                                               <depth unit="mm" cert="medium">75</depth>
+                                           </dimensions>
+                                       </extent>   
+                            <condition key="good">The leather cover on the spine is cut off. 
+                                The manuscript has been resewn. Leather dublours on the inner sides of the boards are
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:57]: What is "dublours"?

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+                           </handNote>                           
+                       </handDesc> 
+                       
+                                                  
+                       <additions>
+                           <list>
+                               <item xml:id="a1">
+                                   <locus from="1ra" to="64b"/>
+                                   <desc type="Unclear">Sālamat-poems</desc>
+                                   <note>Written in a hand similar to the main calligraphic <ref target="#h1">Hand 1</ref>,
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:84]: hand 1

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+                               <item xml:id="a2">
+                                   <locus from="9vb" to="153vb"/>
+                                   <desc type="OwnershipNote">Ownership note, with the name of the owner erased</desc>
+                                   
+                               </item>
+                               
+                               <item xml:id="a3">
+                                   <locus from="9vb" to="153vb"/>
+                                   <desc type="Correction">Numerous corrections throughout the text, made in the margins in tiny
+                                   script with the helf of tamallas and other editorial signs</desc>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:97]: help

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+                                   <note>Written in a hand similar to the main calligraphic <ref target="#h1">Hand 1</ref>,
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:98]: hand 1

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+                                       but inferior in quality, possibly of more recent time.</note>
+                               </item>
+                               
+                               <item xml:id="a4">
+                                   <locus from="154va" to="155va"/>
+                                   <desc type="Unclear">Sālamat-poems, appearing to be 
+                                       the end of Additional note 1, written in the same hand</desc>
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:105]: additional

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 16]
```diff
@@ -0,0 +1,181 @@
...
+                                   <locus from="9vb" to="153vb"/>
+                                   <desc type="Correction">Numerous corrections throughout the text, made in the margins in tiny
+                                   script with the helf of tamallas and other editorial signs</desc>
+                                   <note>Written in a hand similar to the main calligraphic <ref target="#h1">Hand 1</ref>,
+                                       but inferior in quality, possibly of more recent time.</note>
+                               </item>
+                               
+                               <item xml:id="a4">
+                                   <locus from="154va" to="155va"/>
+                                   <desc type="Unclear">Sālamat-poems, appearing to be
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:104]: ሰላማት = Salāmāt ?

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml [hunk 17]
```diff
@@ -0,0 +1,181 @@
...
+                                 </decoNote>
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1700" notAfter="1799">Possibly second half of the 18th century</origDate>.
```
Comment (CarstenHoffmannMarburg) [inline DabraHalleloEndaAbunaSamuel/DHAS012.xml:138]: notBefore="1750"

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: DabraHalleloEndaAbunaSamuel/DHAS012.xml (hunk 13 above)
Comment (DenisNosnitsin1970) [reply DabraHalleloEndaAbunaSamuel/DHAS012.xml:97]: ok

File: DabraHalleloEndaAbunaSamuel/DHAS008.xml [hunk 18]
```diff
@@ -0,0 +1,254 @@
...
+                         
+                         <msPart xml:id="p1">
+                             <msIdentifier>
+                                 <idno/>
+                             </msIdentifier>
+                             <msContents>
+                                 <summary/>            
+                         <msItem xml:id="p1_i1">
+                             <locus from="1ra" to="185vb"/>
+                             <locus target="#196r"/>
```
Comment (nafisa-valieva) [inline DabraHalleloEndaAbunaSamuel/DHAS008.xml:95]: Are you sure the locus target is 196r? It looks confusing, but you might be right.

Comment (DenisNosnitsin1970) [reply DabraHalleloEndaAbunaSamuel/DHAS008.xml:95]: Yes. Unit 2 is inserted just before the last page of Uni 1.

Comment (DenisNosnitsin1970) [reply DabraHalleloEndaAbunaSamuel/DHAS008.xml:95]: But you are right, smth is wrong, I am correcting

File: DabraHalleloEndaAbunaSamuel/DHAS009.xml [hunk 19]
```diff
@@ -0,0 +1,128 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DHAS009" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Miracles of St Mary</title>
```
Comment (nafisa-valieva) [inline DabraHalleloEndaAbunaSamuel/DHAS009.xml:8]: Do we need St.? If yes, probably we need to add St. to everyone considered a saint. But Miracles of Mary sounds more common for me.

Comment (DenisNosnitsin1970) [reply DabraHalleloEndaAbunaSamuel/DHAS009.xml:8]: Not really.

**Review by nafisa-valieva — APPROVED** [review]

**Pull request merged**